flexmeasures/data/scripts/data_gen.py

```diff
@@ -221,51 +220,23 @@ def populate_time_series_forecasts(  # noqa: C901
         name="Seita", type="demo script"
     ).one_or_none()
 
-    # List all old sensors for which to forecast.
-    # Look into their type if no name is given. If a name is given,
-    old_sensors = []
-    if old_sensor_id is None:
-        if old_sensor_class_name is None or old_sensor_class_name == "WeatherSensor":
-            sensors = WeatherSensor.query.all()
-            old_sensors.extend(sensors)
-        if old_sensor_class_name is None or old_sensor_class_name == "Asset":
-            assets = Asset.query.all()
-            old_sensors.extend(assets)
-        if old_sensor_class_name is None or old_sensor_class_name == "Market":
-            markets = Market.query.all()
-            old_sensors.extend(markets)
+    # List all sensors for which to forecast.
+    if sensor_id is None:
+        sensors = Sensor.query.all()
```

Comment:
This is not new, but isn't this dangerous, creating forecasts for every sensor there is? I would consider scrapping this option or think about a better way. Probably this comment can become a new ticket.

- The first reason is that it can amount to too much CPU work, and maybe only because the `--sensor-id` parameter was forgotten. Maybe a conformation dialogue is helpful.
- We now have accounts. If it is actually a feature we want to forecast all available sensors, it should be limitable to one account (at a time)
- Another idea is to forecast an asset (thus all the sensors within that asset), but I'm not sure that is a useful approach.

The safest way forward (imo) is to remove the optionality for now, and see in the future what our actual needs here are.

Reply:
I agree. I implemented a middle ground where the CLI command allows setting multiple `sensor_ids`.